pandas/tests/io/parser/test_network.py

```diff
@@ -52,6 +52,9 @@ def tips_df(datapath):
     return read_csv(datapath("io", "data", "csv", "tips.csv"))
 
 
+s3so = dict(client_kwargs={"endpoint_url": "http://127.0.0.1:5555/"})
```

Comment:
can we make these a fixture so don't have to define everywhere

Reply:
done